website/src/views/DashboardHome/DashboardHometest.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
File Naming

Reply:
I said this above, since there are errors (solved in ur pr, but I cannot get it), Therefore, I keep it like this way and later I just change it. (other wise, the npm test will failed the test)